plugin/nginx_plus_collectd.py

```diff
@@ -787,6 +835,71 @@ def _send_get(self, url):
             LOGGER.exception('Failed request to %s. %s', self.base_status_url, e)
         return status
 
+    def _get_api_type(self):
+        '''
+        Determines whether the Nginx-plus plugin has a 'newer' or 'legacy' version of API.
+        '''
+        legacy_api_url = 'http://{}:{}/status'.format(self.status_host, str(self.status_port))
+        newer_api_url = 'http://{}:{}/api/'.format(self.status_host, str(self.status_port))
+
+        try:
+            response = requests.get(newer_api_url, auth=self.auth_tuple)
+            if response.status_code == requests.codes.ok:
+                return "newer"
```

Comment:
Can you use global variables for `"newer"` and `"legacy"`?

Reply:
Hello @rmfitzpatrick ,

As per the latest design(explained [here](https://github.com/signalfx/collectd-nginx-plus/pull/8#discussion_r757691244)), we are no longer using such notions of `newer` and `legacy`, and hence, no changes are required here.